### PR TITLE
Log reader formater basic

### PR DESCRIPTION
**Pull request by andrzejgorski**

### CODE REVIEW

File: whylog_vim/output_formater/consts.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class LogReader(object):
+    ITEM = 'investigation item'
+    LINE_HEADER = '--- %s [%s offset %s]:'
+    EMPTY_OUTPUT = 'There is no cause of this line in config.'
+    EMPTY_OUTPUT2 = 'To add new rules press whylog teach <F4> by default.'
```
Comment (epawlowska) [inline whylog_vim/output_formater/consts.py]: I suggest: 'To add new rules press whylog teach key (`<F4>` by default).'
Or to know that "whylog teach" is a name: 'To add new rules press WhylogTeach key (`<F4>` by default).'

File: whylog_vim/output_formater/log_reader_formater.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
+from whylog_vim.output_formater.consts import LogReader
+from whylog_vim.output_formater.output_aggregator import OutputAggregator
+
+
+class LogReaderOutput():
```
Comment (epawlowska) [inline whylog_vim/output_formater/log_reader_formater.py]: object...

File: whylog_vim/output_formater/log_reader_formater.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+        )
+        output.add(front_input.line_content)
+
+    @classmethod
+    def format_query(cls, front_input, query_output):
+        output = OutputAggregator()
+        cls._format_input_line(output, front_input)
+        if query_output == []:
+            output.add_commented(LogReader.EMPTY_OUTPUT)
+            output.add_commented(LogReader.EMPTY_OUTPUT2)
```
Comment (epawlowska) [inline whylog_vim/output_formater/log_reader_formater.py]: Maybe give more specific name than `EMPTY_OUTPUT2` ?

File: whylog_vim/output_formater/output_aggregator.py [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+        self.output_lines.append(element)
+
+    def add_commented(self, content):
+        self.add(Messages.COMMENT_PREFIX % content)
+
+    def get_content(self):
+        # The same type as vim buffor.
+        return self.output_lines
+
+    def create_button(self, callback_function, line=None):
```
Comment (epawlowska) [inline whylog_vim/output_formater/output_aggregator.py:19]: Do you really meant `callback`? Shouldn't it be  `call_function`? or `button_call_function`

Comment (andrzejgorski) [inline whylog_vim/output_formater/output_aggregator.py:19]: This is Paweł's proposition.

File: whylog_vim/output_formater/log_reader_formater.py [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+                front_input.offset,
+            )
+        )
+        output.add(front_input.line_content)
+
+    @classmethod
+    def format_query(cls, front_input, query_output):
+        output = OutputAggregator()
+        cls._format_input_line(output, front_input)
+        if query_output == []:
```
Comment (kgromadzki) [inline whylog_vim/output_formater/log_reader_formater.py]: `if not query_output:`

**Pull request merged**